### ISSUE TITLE
parse error

### ISSUE THREAD

**Issue by meeple142** [byuitechops/qualtrics-contact-sync#33]
![image](https://cloud.githubusercontent.com/assets/10274976/24266500/95a7abde-0fcc-11e7-838a-3b776251597f.png)

**Comment by danverde** [comment]
This the the error being thrown:

```
{ Error: Parse Error
    at Error (native)
    at TLSSocket.socketOnData (_http_client.js:363:20)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at TLSSocket.Readable.push (_stream_readable.js:134:10)
    at TLSWrap.onread (net.js:548:20) bytesParsed: 0, code: 'HPE_INVALID_CONSTANT' }
Error: Parse Error
```

**Comment by danverde** [comment]
closed in commit a2e1c0d2b7027f4c0695dac877d041326c7a52ac
